framework/sandstone.h

```diff
@@ -493,12 +529,62 @@ static inline void *aligned_alloc_safe(size_t alignment, size_t size)
     return aligned_alloc(alignment, aligned_size);
 }
 
+/**
+ * @brief If compatibility mode has been requested
+ *
+ * It is allowed to follow old pattern of random number (exact number of bits consumed from
+ * RNG each time), to execute the tests with exactly the same data with previous (pre-100)
+ * suites. For compatibility mode old set_random_bits() is used, otherwise mantissa of floats
+ * is composed from 0s/1s chains (of particular lengths)
+ *
+ * @return true if compatibility mode was requested
+ * @return false to follow new patterns of random data
+ */
+bool random_compatitibilitiy_mode();
```

Comment:
Does this need to be in sandstone.h? That is, does or will any test need this to make decisions? If not, then place place it in sandstone_p.h (private framework API).

If it is meant to stay here, since this is in the C section of the header, add `void` to the parameter list. Also, please move this next to the other random functions (please move `set_random_bits` too, I don't know why `aligned_alloc_safe` is between it and the rest of the random functions) and summarise the docs like the others (no @brief, no @return) - a single sentence starting with "Returns whether" suffices.